crates/next-custom-transforms/tests/fixture/debug-fn-name/1/output.js

```diff
@@ -0,0 +1,14 @@
+function MyComponent() {
+    useLayoutEffect({
+        "MyComponent.useLayoutEffect": ()=>{}
+    }["MyComponent.useLayoutEffect"]);
+    useEffect({
+        "MyComponent.useEffect": ()=>{}
+    }["MyComponent.useEffect"]);
+    const onClick = useCallback({
+        "MyComponent.useCallback": ()=>[]
+    }["MyComponent.useCallback"]);
```

Comment:
This should take the name of the var it's assigned to i.e. 
```suggestion
    }["MyComponent.onClick"]);
```

It's really about the `useCallback` disappearing so that it just looks like you wrote `const onClick = () => {}` on the stack.

Reply:
I updated the PR!